src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.cs

```diff
@@ -63,10 +75,14 @@ private void RevokeListeners()
 	protected override void OnApplyTemplate()
 	{
 		base.OnApplyTemplate();
-
-		_canvas =
-			GetTemplateChild(nameof(PART_Canvas)) as Canvas ??
-			throw new InvalidOperationException($"Canvas '{PART_Canvas}' was not found in the control-template.");
+		_canvas = GetTemplateChild(nameof(PART_Canvas)) as Canvas ??
+			throw new InvalidOperationException($"Grid for Canvas '{PART_Canvas}' was not found in the control-template.");
```

Comment:
In general, controls are not expected to fail when parts of the template cannot be found. Unless the control _cannot function without those parts_, adding debug logs for this would probably be best.